bin/loading-page.html

```diff
@@ -73,7 +73,7 @@ <h4 class="Footer-links-header"><a href="/en-GB/firefox/"><!-- react-text: 236 -
                 </a>
               </li>
               <li>
-                <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies
+                <a href="https://extensionworkshop.com/documentation/publish/add-on-policies/">Developer Policies
```

Comment:
No need to add the query params here.

Reply:
Yeah I did not, though in below lines they are added, in browser links